charts/fluid/fluid/templates/role/csi/rbac.yaml

```diff
@@ -0,0 +1,117 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: fluid-csi-admin
+  namespace: fluid-system
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: fluid-csi-plugin
+rules:
+  - apiGroups:
+      - data.fluid.io
+    resources:
+      - alluxiodataloads
```

Comment:
Do we need access alluxiodataloads? I think maybe jindoRuntime, GooseRuntime is required. We should consider seriously about if the resource access is required.

Reply:
Fixed. The RBAC has been minimized.